drivers/hmis/lib/form_data/default/fragments/w4_w6_hopwa_disability.json

```diff
@@ -36,7 +36,6 @@
       "link_id": "W4_A",
       "required": false,
       "text": "T-Cell Count",
-      "pick_list_reference": "NoYesReasonsForMissingData",
```

Comment:
pick_list_reference on an integer doesn't make sense(?)

Reply:
yep this gets ignored